### DISCUSSION TITLE
psql.X missing?

### DISCUSSION THREAD

**Discussion by RangelReale** [stephenafamo/bob#59] [General]
Following the documentation, it says there is a starter `X()` for all dialects, but I don't seem to find `psql.X`, was this removed?

**Comment by stephenafamo** [comment]
The documentation is outdated. I thought I had changed it but it seems I missed a spot.

The `X()` starter was removed in the [`v0.18.0` release](https://github.com/stephenafamo/bob/releases/tag/v0.18.0)

**Reply by RangelReale** [reply]
I need to do this code, is there any other way besides raw code?

```go
sm.Having("count(*) > 1"),
```

**Reply by stephenafamo** [reply]
You have 2 options:

```go
sm.Having(psql.Raw("count(*) > 1")),
```

OR

```go
sm.Having(psql.F("count", "*").GT(1)),
```
